platforms/core-execution/persistent-cache/src/integTest/groovy/org/gradle/cache/internal/DefaultFileLockManagerContentionIntegrationTest.groovy

```diff
@@ -90,13 +88,16 @@ class DefaultFileLockManagerContentionIntegrationTest extends AbstractIntegratio
         when:
         def build = executer.withTasks("lock").start()
         def timer = Time.startTimer()
+
+        def pingCountInOutput = 0
         poll(120) {
-            assert (build.standardOutput =~ 'Pinged owner at port').count == 3
+            pingCountInOutput = (build.standardOutput =~ 'Pinged owner at port').count
+            assert pingCountInOutput >= 3
         }
         receivingLock.close()
         then:
         build.waitForFinish()
-        pingRequestCount == 3 || pingRequestCount == 4
+        pingRequestCount in [pingCountInOutput, pingCountInOutput + 1]
```

Comment:
💅 do we need an "exact" match here? or would a `pingRequestCount >= pingCountInOutput` be sufficiently good?